lib/fizzy/execute.cpp

```diff
@@ -173,7 +172,7 @@ std::tuple<table_ptr, Limits> allocate_table(
 }
 
 std::tuple<bytes_ptr, Limits> allocate_memory(const std::vector<Memory>& module_memories,
-    const std::vector<ExternalMemory>& imported_memories)
+    const std::vector<ExternalMemory>& imported_memories, unsigned memory_pages_limit)
```

Comment:
```suggestion
    const std::vector<ExternalMemory>& imported_memories, uint32_t memory_pages_limit)
```
Always use uint32_t for memory indexing and sizes?

Reply:
This is number of pages, the limits sections uses `uint32_t`, and we actually use `size_t` in multiple places regarding this.